guides/tutorial/start-tutorial.md

```diff
@@ -1,3 +1,384 @@
-# not yet implemented
+# Getting started with Guardian
 
-if you'd like to help with this guide please submit a pull request to (guardian)[https://github.com/ueberauth/guardian] in the `guides/tutorial/start-turorial.md` file.
+Getting started with Guaridan is easy. This tutorial will cover
+
+* Setting up the basics
```

Comment:
These bullets are not all that clear...what does "Setting up the basics" mean?

"HTTP based setup" is also confusing to me. The word "setup" seems to be what we did in the first bullet point?